dali/test/python/test_pytorch_operator.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import os
+import random
+import torch
+import nvidia.dali.plugin.pytorch as dalitorch
+import numpy
+
+
+test_data_root = os.environ['DALI_EXTRA_PATH']
+images_dir = os.path.join(test_data_root, 'db', 'single', 'jpeg')
+
+
+def random_seed():
+    return int(random.random() * (1 << 32))
+
+
+DEVICE_ID = 0
+BATCH_SIZE = 8
+ITERS = 64
+SEED = random_seed()
+NUM_WORKERS = 6
+
+
+class CommonPipeline(Pipeline):
+    def __init__(self, batch_size, num_threads, device_id, _seed, image_dir):
+        super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, seed=_seed,
+                                             exec_async=False, exec_pipelined=False)
+        self.input = ops.FileReader(file_root=image_dir)
+        self.decode = ops.HostDecoder(output_type=types.RGB)
+
+    def load(self):
+        jpegs, labels = self.input()
+        decoded = self.decode(jpegs)
+        return decoded, labels
+
+    def define_graph(self):
```

Comment:
you can remove this

Reply:
Yes, because the operator is registered during the module initialization.